easybuild/easyconfigs/s/Sambamba/Sambamba-0.6.6.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'Tarball'
+
+name = 'Sambamba'
+version = '0.6.6'
+
+homepage = 'http://lomereiter.github.io/sambamba/'
+description = """Sambamba is a high performance modern robust and fast tool (and library), 
+ written in the D programming language, for working with SAM and BAM files. Current functionality 
+ is an important subset of samtools functionality, including view, index, sort, markdup, and depth"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/lomereiter/sambamba/releases/download/v%(version)s/']
+sources = ['%(namelower)s_v%(version)s_linux.tar.bz2']
+
+sanity_check_paths = {
+    'files': ['sambamba_v%(version)s'],
+    'dirs': [],
+}
+
+modextrapaths = {
+    'PATH': "",
+}
+
+postinstallcmds = ['cd %(installdir)s && ln -s sambamba_v%(version)s sambamba',]
```

Comment:
@pescobar please move this up, above `sanity_check_paths`, and remove the trailing `,`